pkg/model/manifest.go

```diff
@@ -112,8 +112,8 @@ func (m Manifest) IsK8s() bool {
 	return ok
 }
 
-func (m Manifest) IsUnresourcedYAMLManifest() bool {
-	return m.Name == UnresourcedYAMLManifestName
+func (m Manifest) IsNonWorkloadYAMLManifest() bool {
```

Comment:
a. hannnng on, this would tell us that e.g. a local resource manifest or a docker-compose-no-image manifest was a non-workload manifest, that seems wrong. Also
b. you can have a workload manifest that we don't build images for! (i.e. a manifest that spins up pods running an image pulled from the internet.)
c. less importantly, "NonWorkloadYAMLManifest" seems redundant, couldn't you just say "NonWorkloadManifest"? (i.e. IF it's a k8s manifest and is non-workload, it'll necessarily be a yaml-only manifest)

Reply:
Okay, I'm going to back to the strategy of setting a bit on the k8sTarget during Tiltfile execution before things get passed off to the engine, and then check that bit here.

Reply:
yay i like this a lot better!

one nit: "NonWorkloadYAMLManifest" seems redundant, couldn't you just say "NonWorkloadManifest"? (i.e. IF it's a k8s manifest and is non-workload, it'll necessarily be a yaml-only manifest)